client/src/components/VacaturLawsKeyModal.jsx

```diff
@@ -3,132 +3,142 @@ import "../styles/UploadModal.css";
 
 const modalActiveClass = "pt-4 px-2 text-xl inline font-bold";
 const categoryHeader =
-    "pt-2 px-2 text-base inline-block font-bold align-middle";
+  "pt-2 px-2 text-base inline-block font-bold align-middle";
 const categoryDescription = "pt-1 px-2 text-sm inline";
 
 const VacaturLawsKeyModal = (props) => {
-    const { closeModal, modalVisible } = props;
+  const { closeModal, modalVisible } = props;
 
-    const onClose = (e) => {
-        e.stopPropagation();
-        closeModal();
-    };
+  const onClose = (e) => {
+    e.stopPropagation();
+    closeModal();
+  };
 
-    return (
-        <>
-            {modalVisible && (
-                <div className="modal" onClick={closeModal}>
-                    <div className="modal-content" onClick={(e) => e.stopPropagation()}>
-                        <span className="close" onClick={onClose}>
-                            &times;
+  return (
+    <>
+      {modalVisible && (
+        <div className="modal" onClick={closeModal}>
+          <div className="modal-content" onClick={(e) => e.stopPropagation()}>
+            <span className="close" onClick={onClose}>
+              &times;
             </span>
-                        <div class="flex">
-                            <p class={modalActiveClass}>Vacatur Laws Key</p>
-                        </div>
-                        <div class="flex">
-                            <div class={categoryHeader}>
-                                <div className="square goodCategory"></div>Excellent
+            <div class="flex">
+              <p class={modalActiveClass}>Vacatur Laws Key</p>
+            </div>
+            <div class="flex">
+              <div class={categoryHeader}>
+                <div className="square goodCategory"></div>Excellent
               </div>
-                        </div>
-                        <div class="flex">
-                            <p class={categoryDescription}>
-                                These states provide robust avenues for civil relief that are 
-                                trauma-informed and allow survivors of trafficking to live free 
-                                of documentation that records them as criminals rather than survivors
+            </div>
+            <div class="flex">
+              <p class={categoryDescription}>
```

Comment:
Not entirely sure what's making this happen but the `categoryHeader` class is also being applied to the description text.

![image](https://user-images.githubusercontent.com/10099821/103381471-79ea5800-4ab1-11eb-8838-baf85656c723.png)

Reply:
Should be fixed now, was because it was inside a component in Map.jsx which it was inheriting from